### ISSUE TITLE
Disconnected

### ISSUE THREAD

**Issue by RaphHal0** [games647/ChangeSkin#164]
[//]: # (Lines in this format are considered as comments and will not be displayed.)
[//]: #
[//]: # (Before reporting an issue make sure you are running the latest build and checked for duplicate issues!)

### What behaviour is observed:
[//]: # When i choose a skin, i have been disconnected of my server :/

### What behaviour is expected:
[//]: # I don't know what is the problem in that

### Steps/models to reproduce:
[//]: # I write /skin (name)

### Plugin list:
[//]: # essentials, vault, Luckperms, MinePacks, OpenInv, ClearLag, TrollBoss, SimplSit, Crazyenchantment, chestshop, mythicMobs, Jobs, QuickMarket and Changeskin

### Environment description
[//]: # I use Catserver in 1.12.2

### Plugin version or build number:
[//]: # version 3.0
[//]: # (This can be found by running `/version plugin-name`.)

### Error Log:
There is no error it's sad

### Configuration:
[//]: # (Remember to delete any sensitive data)
[Hastebin](https://hastebin.com/) / [Gist](https://gist.github.com/) link of your config.yml file

**Comment by games647** [comment]
Do you use BungeeCord? If you don't have ProtocolLib installed, you have to re-login for changes of skin. So it seems like a bug your server software.

**Comment by RaphHal0** [comment]
No, i don’t use BungeeCord and i have protocolLib. Its very interesting :/

Provenance : Courrier pour Windows 10

De : games647
Envoyé le :14 juin 2019 03:21
À : games647/ChangeSkin
Cc : RaphHal0; Author
Objet :Re: [games647/ChangeSkin] Disconnected (#164)

Do you use BungeeCord? If you don't have ProtocolLib installed, you have to re-login for changes of skin. So it seems like a bug your server software.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.



---
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Comment by games647** [comment]
Do you use a vanilla Minecraft client?

**Comment by RaphHal0** [comment]
No, I use a server with mods + plugins

Provenance : Courrier pour Windows 10

De : games647
Envoyé le :20 juin 2019 15:04
À : games647/ChangeSkin
Cc : RaphHal0; Author
Objet :Re: [games647/ChangeSkin] Disconnected (#164)

Do you use a vanilla Minecraft client?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.



---
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus

**Comment by games647** [comment]
Then instant skin changes properly doesn't work. You could disable this feature in the configuration. Then you have to relogin for the changes to apply.

**Comment by RaphHal0** [comment]
But before, it worked, what I have to disable in the configuration ? I need to test if the plugin don’t work properly with another plugin.

Provenance : Courrier pour Windows 10

De : games647
Envoyé le :21 juin 2019 04:55
À : games647/ChangeSkin
Cc : RaphHal0; Author
Objet :Re: [games647/ChangeSkin] Disconnected (#164)

Then instant skin changes properly doesn't work. You could disable this feature in the configuration. Then you have to relogin for the changes to apply.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.



---
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus
